### ISSUE TITLE
clang doesn't know about -Wmisleading-indentation

### ISSUE THREAD

**Issue by llvmbot** [llvm/llvm-project#24284]
|  |  |
| --- | --- |
| Bugzilla Link | [23910](https://llvm.org/bz23910) |
| Version | unspecified |
| OS | FreeBSD |
| Reporter | LLVM Bugzilla Contributor |
| CC | @emaste,@gnzlbg,@sylvestre |

**Comment by Quuxplusone** [comment]
This warning has been introduced in Clang 10.
```
<source>:9:9: warning: misleading indentation; statement is not part of the previous 'if' [-Wmisleading-indentation]
        goto derp;
        ^
```
